packages/sdk-router/src/typechain/FastBridgeInterceptor.ts

```
@@ -0,0 +1,206 @@
+    data: BytesLike
+  ): Result
+  decodeFunctionResult(
+    functionFragment: 'NATIVE_GAS_TOKEN',
+    data: BytesLike
+  ): Result
+  decodeFunctionResult(
+    functionFragment: 'bridgeWithInterception',
+    data: BytesLike
+  ): Result
+
+  events: {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)